packages/eslint-config/index.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     "plugin:testing-library/react",
     "plugin:jest-dom/recommended",
     "plugin:jsx-a11y/recommended",
+    "plugin:tailwindcss/recommended",
```

Comment:
Unrelated, can you add a comment above the `prettier` line?

> Prettier must be the last extension in the list. Prettier works best if you disable all other ESLint rules relating to code formatting, and only enable rules that detect potential bugs. (If another active ESLint rule disagrees with prettier about how code should be formatted, it will be impossible to avoid lint errors.)